src/app/contexts/Ledger/use-ledger.ts

```diff
@@ -32,6 +34,20 @@ export const useLedger = (transport: typeof Transport) => {
 		[transport],
 	);
 
+	const importLedgerWallets = useCallback(
+		async (wallets: { address: string; index: number }[], coin: Coins.Coin, profile: Profile) => {
+			for (const { address, index } of wallets) {
+				const wallet = await profile
+					.wallets()
+					.importByAddress(address, coin.network().coin(), coin.network().id());
+				wallet.data().set(WalletFlag.Ledger, true);
+				wallet.data().set("LEDGER_INDEX", index);
```

Comment:
Add a `WalletFlag` to the SDK for this or this could get removed because the data is unknown.

Reply:
Just used the current flag to set the index, looks ok or better to add a new one?

Reply:
Should work though an explicit `LedgerIndex` attribute makes it more clear what it is and what it could be used for.

Reply:
https://github.com/ArkEcosystem/platform-sdk/releases/tag/0.9.393